tests/test_fields.py

```diff
@@ -9,7 +9,12 @@
 from unittest.mock import patch
 
 import pytest
-import pytz
+
+try:
+    import pytz
+except ImportError:
+    pytz = None
```

Comment:
I added this, so it's possible the `@pytest.mark.skipif(pytz is None)` on [line 1598](https://github.com/encode/django-rest-framework/pull/8984/files#diff-a828467ae280cb3ae472f4c3fa7fafe22f9e672829114f1ecccc14435fa205f8R1598) never worked